_config.yml

```diff
@@ -100,6 +92,61 @@ gh_edit_view_mode: "tree" # "tree" or "edit" if you want the user to jump into t
 # Color scheme currently only supports "dark", "light"/nil (default), or a custom scheme that you define
 color_scheme: nil
 
+# Callouts
+# 
+# The title of a callout is optional. It is rendered in uppercase.
+# 
+# Each kind of callout has a color, used for the callout bar, the title, and the background.
+# Predefined colors: grey-lt, grey-dk, purple, blue, green, yellow, red.
+# If you use another color, you should define its levels (000, 100, 200, 300) in _sass/custom/custom.scss.
+#
+# The default is for callout backgrounds to have a lower color level than the bars and titles.
+# The value of `callouts_level` is either `quiet` or `loud`:
+# `loud` increases the saturation and lightness of the backgrounds.
+# The default for `callouts_level` is `quiet` when using the `light` or custom color schemes,
+# and `loud` when using the `dark` color scheme.
+# 
+# Examples (not provided by default):
+# 
+# callouts:
+#   admonition:
+#     # title:
+#     color: grey-lt
+#   attention:
+#     title: Attention
+#     color: purple
+#   caution:
+#     title: Caution
+#     color: yellow
+#   custom:
+#     title: Custom
+#     color: pink
+#     opacity: 0.3
+#   danger:
+#     title: Danger
+#     color: red
+#   error:
+#     title: Error
+#     color: red
+#   highlight:
+#     color: yellow
+#     opacity: 0.5
+#   hint:
+#     title: Hint
+#     color: grey-dk
+#   important:
+#     title: Important
+#     color: green
+#   note:
+#     title: Note
+#     color: blue
+#   tip:
+#     title: Tip
+#     color: grey-dk
+#   warning:
+#     title: Warning
+#     color: red
```

Comment:
I think they should *all* be removed from the config! The explanations in https://github.com/pdmosses/just-the-docs/blob/combination/docs/ui-components/callouts.md and https://github.com/pdmosses/just-the-docs/blob/combination/docs/configuration.md#callouts should be adequate (or improved if they aren't, although that may have to be deferred to a later version).

Perhaps the same principle should apply to all the other configurable elements too: remove the commented-out options from `_config.yml`. The theme docs shouldn't need to refer to the config file used to produce the website, except  maybe in a (new) section aimed at advanced users who are going to base a new theme on this one.

For now, I'm removing from `_config.yml` only the cited commented-out lines regarding callouts. Pruning comments from the rest of the config file should be discussed separately.

However, note that to illustrate the effect of callout config options in the theme web pages, we'd need to add them (*not* commented-out) in the theme config.

BTW, I think the following comment at the beginning of https://just-the-docs.github.io/just-the-docs/docs/configuration/ is particularly confusing for new users:

> View this site’s [_config.yml](https://github.com/just-the-docs/just-the-docs/tree/main/_config.yml) file as an example.

New users should generally start with a much simpler config. Perhaps the new repo for Just the Docs tests could have an example of such a simple config, and the theme docs could link to that.

Reply:
Understood, thank you for the response.

> Perhaps the same principle should apply to all the other configurable elements too: remove the commented-out options from _config.yml. 

Hm, that's an interesting take. I personally dislike it when config files get too big with commented-out examples, but I'm also concerned from a usability perspective - I do know many of our users don't have significant background in web development, configs, etc.

> New users should generally start with a much simpler config. Perhaps the new repo for Just the Docs tests could have an example of such a simple config, and the theme docs could link to that.

I like the suggestion, especially wrt #756.

Reply:
From a usability perspective, I think the best for all users is to read _explanations_ and _illustrations_ of the various configuration options in the [Configuration](https://just-the-docs.github.io/just-the-docs/docs/configuration/) section of the theme docs, rather than expecting them to browse the raw `_config.yml` file, where some of the settings are not intended for general use. Not all the options are currently explained.

Moreover, I think we should consider dividing the current [Configuration](https://just-the-docs.github.io/just-the-docs/docs/configuration/) page into a page for each feature (or group of related features, such as customising the content of the different parts of pages). But that can wait to a later release.

Reply:
> But that can wait to a later release.

We could also improve the theme docs _between_ releases: a PR to fix typos or improve navigation on the theme website can be merged into `main` as soon as it has been approved, without triggering a new release.